src/main/java/org/folio/circulation/domain/representations/DeclareItemLostRequest.java

```diff
@@ -38,4 +38,8 @@
     return succeeded(new DeclareItemLostRequest(loanId, dateTime, comment,
       getProperty(json, "servicePointId")));
   }
+
+  public String getServicePointId() {
+    return this.servicePointId;
+  }
```

Comment:
Given this property is now required, should the process fail if none is provided?

Reply:
I was honestly unsure if that was something my code needed to handle-I haven't been working with RAML very long, so I wasn't sure if it would automatically fail requests without parameters listed as required-it does an awful lot of other stuff automatically that I wouldn't have expected.  Yes, it should fail, and of course, now that I'm thinking about it, I can see it *does* check for the date data, and fails it if not valid, so of course it actually needs a check.  I'll put one in.

Reply:
> I was honestly unsure if that was something my code needed to handle-I haven't been working with RAML very long, so I wasn't sure if it would automatically fail requests without parameters listed as required - it does an awful lot of other stuff automatically that I wouldn't have expected.

That varies by the tools being used. RAML Module Builder does this and I think Spring Way does too. As mod-circulation uses neither of those, it does not do this automatically. There is some code that uses the schema to do this, it isn't used very widely.

> Yes, it should fail, and of course, now that I'm thinking about it, I can see it _does_ check for the date data, and fails it if not valid, so of course it actually needs a check. I'll put one in.

Thanks